tests/test-projects/basic/cypress/integration/readOnly_spec.js

```diff
@@ -9,7 +9,9 @@ describe('ReadOnly Fields', () => {
 
     cy.get('a[href^="/admin/read-only-lists/"]:first').click({ force: true });
 
-    ['slug', 'status', 'author', 'views', 'price', 'currency', 'hero'].forEach(field => {
+    // FIXME: Add price when we have Decimal support in prisma
```

Comment:
@timleslie is this an issue that exists in our backlog, if not can we add it (or add it to an existing issue)

Reply:
I have a WIP branch which is following the development in Prisma. I'll add an issue for it 👍